app.js

```diff
@@ -136,146 +164,129 @@ var processRequest = function (req, res, isTar) {
+    const opts = {
       url: global.LDAP_URL,
       bindDn: global.LDAP_BIND_DN.replace(/{{username}}/, username),
       bindCredentials: password,
       searchBase: global.LDAP_SEARCH_BASE,
       searchFilter: '(uid={{username}})',
       searchAttributes: ['memberOf'],
-      cache: false
+      cache: false,
     };
 
-    var ldap = new LdapAuth(opts);
+    const ldap = new LdapAuth(opts);
 
-    ldap.authenticate(username, password, function (err, user) {
+    ldap.authenticate(username, password, (err, user) => {
       if (err) {
+        // eslint-disable-next-line no-console
         console.log('LDAP auth error: %s', err);
       }
       done(null, user);
     });
-  }
-));
+  }),
+);
 
-app.post('/api/request', function (req, res, next) {
-  if (req.is('application/x-www-form-urlencoded')) { // URL + token method
+app.post('/api/request', (req, res, next) => {
+  if (req.is('application/x-www-form-urlencoded')) {
+    // URL + token method
     processRequest(req, res, false);
-  }
-  else if (req.is('multipart/form-data')) { // tar method
+  } else if (req.is('multipart/form-data')) {
+    // tar method
     next();
-  }
-  else {
-    res.status(501)
-       .send({ status: 'Form Content-Type not supported' });
+  } else {
+    res.status(501).send({ status: 'Form Content-Type not supported' });
   }
 });
 
-app.post(
-  '/api/request',
-  multer().single('tar'),
-  function (req, res, next) {
-    if (req.headers.authorization) { // basic authentication
-      next();
-    } else {
-      processRequest(req, res, true); // token
-    }
+app.post('/api/request', multer().single('tar'), (req, res, next) => {
+  if (req.headers.authorization) {
+    // basic authentication
+    next();
+  } else {
+    processRequest(req, res, true); // token
   }
-);
+});
 
 app.post(
   '/api/request',
   passport.authenticate('basic', { session: false }),
-  function (req, res) {
+  (req, res) => {
     processRequest(req, res, true);
-  }
+  },
 );
 
-/**
-* Add CORS headers to responses if the client is explicitly allowed.
-*
-* First, this ensures that the testbed page on the test server, listening on a different port, can GET and POST to Echidna.
-* Most importantly, this is necessary to attend publication requests from third parties, eg GitHub.
-*/
-
-function corsHandler(req, res, next) {
-  if (req && req.headers && req.headers.origin) {
-    if (global.ALLOWED_CLIENTS.some(function (regex) {
-      return regex.test(req.headers.origin);
-    })) {
-      res.header('Access-Control-Allow-Origin', req.headers.origin);
-      res.header('Access-Control-Allow-Methods', 'GET,POST');
-      res.header('Access-Control-Allow-Headers', 'Content-Type');
-    }
-  }
-  next();
-}
-
-app.listen(process.env.PORT || port).on('error', function (err) {
+app.listen(process.env.PORT || port).on('error', err => {
   if (err) {
-    console.error('Error while trying to launch the server: “' + err + '”.');
+    // eslint-disable-next-line no-console
```

Comment:
We can probably add the `no-console` in the global ESLint config. We do make use of this quite some time.